backend/.env.test

```
@@ -0,0 +1,17 @@
+PORT=3011
+IP=127.0.0.1
+JWT_SECRET=dont_use_this_in_production
+PORT=3011
+IP=127.0.0.1
+JWT_SECRET=dont_use_this_in_production
+# Make sure this is higher than the test takes to run.
+JWT_EXPIRATION=30m
+JWT_REFRESH_SECRET=dont_use_this_in_production2
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_REFRESH_SECRET key should go before the JWT_SECRET key

(UnorderedKey)